src/wp-includes/class-wp-theme-json.php

```diff
@@ -394,18 +394,19 @@ class WP_Theme_JSON {
 	/**
 	 * Defines which pseudo selectors are enabled for which elements.
 	 *
-	 * The order of the selectors should be: visited, hover, focus, active.
-	 * This is to ensure that 'visited' has the lowest specificity
-	 * and the other selectors can always overwrite it.
+	 * The order of the selectors should be: link, any-link, visited, hover, focus, active.
+	 * This is to ensure that 'visited' has the lowest specificity and the other selectors
+	 * can always overwrite it.
```

Comment:
This might need a little work.

```suggestion
	 * This is to ensure the user action (hover, focus and active) styles have a
	 * higher specificity than the visited styles, which in turn have a higher 
	 * specificity than the unvisited styles.
```